controllers/recipeController.js

```diff
@@ -62,27 +62,82 @@ recipeController.createRecipe = (req, res, next) => {
             fs.unlink(tempPath, () => {
 
               var recipeObject = new recipeModel(
-                fields.title[0], 
-                fields.description[0], 
-                false, 
+                fields.title[0],
+                fields.description[0],
+                false,
                 Number(fields.owner_id[0]),
                 fullPath,
                 Number(fields.rating[0])
               );
 
               db.query(recipeObject.saveRecipe(recipeObject), (err, result) => {
                 if (err) {
-                  return next(err); 
+                  return next(err);
                 } else {
                   res.send("Recipe created!");
                 }
               });
             });
-          }); 
-        }); 
-      } 
+          });
+        });
+      }
     }
   });
 };
 
+
+const recipeHelper = (dbResponse) => {
+    const ingredients = [];
+
+    dbResponse.forEach((field) => {
+        ingredients.push(field.name);
+    });
+
+    const recipe = {
+        id: dbResponse[0].id,
+        title: dbResponse[0].title,
+        description: dbResponse[0].description,
+        rating: dbResponse[0].rating,
+        photo: dbResponse[0].photo,
+        ingredients,
+    };
+
+    return recipe;
+};
+
+recipeController.getRecipeById = (req, res) => {
+    const id = req.params.id;
+
+    const isPositiveIntegerNumeric = (arg) => {
+        if (isNaN(+arg)) return false;
```

Comment:
o_O why not just `isNumber` from lodash? Or you need to convert string to number?